code/game/turfs/simulated/floor/lava.dm

```diff
@@ -1,8 +1,10 @@
-/turf/simulated/floor/plating/lava
+/turf/simulated/floor/lava
```

Comment:
please add some general documentation to this type. Explain why we relegate this to it's own floor subtype.